source/_integrations/filter.markdown

```diff
@@ -61,7 +60,7 @@ entity_id:
   required: true
   type: string
 name:
-  description: Name to use in the frontend.
+  description: the name is used to create the new entity_id and is also used as the friendly_name of the new entity. 
```

Comment:
This is not unique to this integration/platform, but the default behavior of Home Assistant. Therefore documenting it here specifically, but not on all other occurrences is weird.

I suggest not to add this here and revert this change.

```suggestion
  description: Name to use in the frontend.
```